airbyte-webapp/src/test-utils/testutils.tsx

```diff
@@ -49,8 +49,8 @@ export async function render<
   };
 
   let renderResult: RenderResult<Q, Container>;
-  await act(async () => {
-    renderResult = await rtlRender<Q, Container>(<div>{ui}</div>, { wrapper: Wrapper, ...renderOptions });
+  act(() => {
+    renderResult = rtlRender<Q, Container>(<div>{ui}</div>, { wrapper: Wrapper, ...renderOptions });
```

Comment:
This likely needed the await

Reply:
It needed the async for the `act` function, but the await for rtlRender was unnecessary, since that does not return a promise.